int/api/myplugin/register.go

```diff
@@ -20,13 +24,32 @@ func (r *register) Handle(param operations.PluginManagerRegisterParams) middlewa
 		return operations.NewPluginManagerRegisterNotFound()
 	}
 
+	urlPlugin, err := url.Parse(param.Body.URL)
+	if err != nil {
+		return operations.NewPluginManagerRegisterBadRequest().WithPayload(
+			&models.Error{Code: errorCodePluginRegisterInvalidData, Message: fmt.Sprintf("parsing URL: %s", err.Error())},
+		)
+	}
+
+	// Set plugin information.
 	info := plugin.Information{
-		Name: param.Body.Name, Description: param.Body.Description,
-		Logo:      param.Body.Logo,
-		Authority: param.Body.Authority, APISpec: param.Body.APISpec,
+		Name: param.Body.Name, Author: param.Body.Author,
+		Description: param.Body.Description,
+		Logo:        param.Body.Logo,
+		URL:         urlPlugin, APISpec: param.Body.APISpec,
```

Comment:
Why not putting Author and APISpec on their own line ? Would be easier to read I think

Reply:
They are regrouped because both concern the remote API (plugin API).